apps/system/lockscreen/js/lockscreen.js

```diff
@@ -560,6 +568,7 @@
       this._lastUnlockedInterval = now - this._lastUnlockedTimeStamp;
       this._lastLockedTimeStamp = now;
 
+      this.playUnlockSound('/resources/sounds/lock.opus');
```

Comment:
Well to change it like that will be a complicated patch, although that's indeed better. I hope this can still follow the original (although it's legacy) flow similar with how we play unlocking sound in this file. If the `details` is missing in some cases, I think it should be fixed.